content/instances/instances.md

```diff
@@ -0,0 +1,15 @@
+---
+date: "2018-06-06T16:00:00+02:00"
+title: "Instances"
+weight: 35
+toc: true
+draft: false
+url: "instances"
+type: "page"
+---
+
+| Instance-Name                                     | TLS/SSH | IPv6/IPv4 | Free User (public/private) | Free Orgs (public/private) | Limits or Rules |
+|---------------------------------------------------|---------|-----------|----------------------------|----------------------------|--------------|
+| [try.gitea.io](https://try.gitea.io)              |  ✔ / ✔  |   ✔ / ✔   |                ✔ / ✔   |                    ✔ / ✔   | temporary/delete by version-upgrade |
+| [git.spacegit.xyz](https://git.spacegit.xyz)      |  ✔ / ✔  |   ✔ / ✔   |                ✔ / ✘   |                    ✘ / ✘   | [see](https://git.spacegit.xyz/)   |
```

Comment:
git.spacegit.xyz offers free private hosting as well, and orgs. (orgs by request). However if users want us to host a different install for them then we charge.